services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/CongestionTest.java

```diff
@@ -0,0 +1,49 @@
+package com.mapbox.api.directions.v5.models;
+
+import com.mapbox.core.TestUtils;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class CongestionTest extends TestUtils {
+
+  @Test
+  public void sanity() {
+    assertNotNull(getDefault());
+  }
+
+  @Test
+  public void testSerializable() throws Exception {
+    Congestion congestion = getDefault();
+    byte[] serialized = TestUtils.serialize(congestion);
+
+    assertEquals(congestion, deserialize(serialized, Congestion.class));
+  }
+
+  @Test
+  public void jsonTest1() {
+    Congestion congestion = getDefault();
+    String json = congestion.toJson();
+
+    Congestion fromJson = Congestion.fromJson(json);
+
+    assertEquals(congestion, fromJson);
+  }
+
+  @Test
+  public void jsonTest2() {
```

Comment:
Same as above

> Minor thing: Could we add a better naming reflecting what's being tested here?

Reply:
https://github.com/mapbox/mapbox-java/pull/1201#discussion_r502018321

Reply:
renamed to `jsonComparingCustomCongestion`